spec/core/cbp_helper.rb

```diff
@@ -0,0 +1,21 @@
+def expect_cbp_response_for(collection)
+  context collection.path.to_s, :vcr do
+    before do
+      @resource_klass = collection.instance_variable_get(:@resource_class)
```

Comment:
Any way around poking inside the private variables of classes?

Reply:
yes, this was taken care of